demo-projects/blog/package.json

```diff
@@ -33,7 +33,9 @@
     "@keystonejs/keystone": "^6.0.1",
     "@keystonejs/oembed-adapters": "^5.1.2",
     "apollo-boost": "^0.4.7",
+    "apollo-cache-inmemory": "^1.6.5",
     "apollo-client": "^2.6.8",
+    "apollo-link-http": "^1.5.16",
```

Comment:
I can't see why this has been added to the dependencies?

Reply:
Good catch!